etna/home/migrations/0016_alter_homepage_body.py

```diff
@@ -0,0 +1,99 @@
+                                ),
+                                (
+                                    "page",
+                                    wagtail.blocks.PageChooserBlock(
+                                        page_type=[
+                                            "collections.TimePeriodExplorerIndexPage"
+                                        ]
+                                    ),
+                                ),
+                            ]
+                        ),
+                    ),
+                    (
+                        "topic_explorer",
+                        wagtail.blocks.StructBlock(
+                            [
+                                (
+                                    "heading",
+                                    wagtail.blocks.CharBlock(
+                                        default="Explore by topic", max_length=100
+                                    ),
+                                ),
+                                (
+                                    "page",
+                                    wagtail.blocks.PageChooserBlock(
+                                        page_type=["collections.TopicExplorerIndexPage"]
+                                    ),
+                                ),
+                            ]
+                        ),
+                    ),
+                    (
+                        "paragraph",
+                        wagtail.blocks.StructBlock(
+                            [
+                                (
+                                    "text",
+                                    wagtail.blocks.RichTextBlock(
+                                        features=["bold", "italic", "link", "ul"]
+                                    ),
+                                )
+                            ]
+                        ),
+                    ),
+                    (
+                        "paragraph_with_heading",
+                        wagtail.blocks.StructBlock(
+                            [
+                                (
+                                    "heading",
+                                    wagtail.blocks.CharBlock(
+                                        max_length=100, required=True
+                                    ),
+                                ),
+                                (
+                                    "paragraph",
+                                    wagtail.blocks.RichTextBlock(
+                                        features=["bold", "italic", "link", "ul"],
+                                        required=True,
+                                    ),
+                                ),
+                            ]
+                        ),
+                    ),
+                ],
+                blank=True,
+                null=True,
+                use_json_field=True,
+            ),
+        ),
+    ]
```

Comment:
Since all the above changes in the home page, 15 and 16, can we combine them in a single migration?

Reply:
I'm not sure how to do this one mate I just type makemigrations and let the magic work 😂

Reply:
No problem, you need to delete these two migrations and then run makemigration , Django will notice the change and make a new migration to include all.